cpu/esp32/Kconfig

```diff
@@ -93,4 +100,22 @@ config CPU_MODEL
 config CPU
     default "esp32" if CPU_FAM_ESP32
 
+menu "ESP32 configurations"
```

Comment:
Not part of this PR. Do you mind to move
```
config HAS_ESP_WIFI_ENTERPRISE
    bool
    help
        Indicates that the platform supports WPA2 enterprise mode for the WiFi
        interface.
```
to `cpu/esp_common/Kconfig`? It's not a special feature of ESP32 and with a newer ESP8266 it will also be a feature of ESP8266. So the definition of the feature should be in `esp_common` and the CPU should only select the feature.

Reply:
Ok, I moved it in a2700de6ad7dc391317c6d7dd159ae99781cde81